esp-hal/src/rtc_cntl/rtc/esp32c6.rs

```diff
@@ -1361,7 +1361,7 @@ pub(crate) enum RtcFastClock {
 impl Clock for RtcFastClock {
     fn frequency(&self) -> HertzU32 {
         match self {
-            RtcFastClock::RtcFastClockXtalD2 => HertzU32::Hz(40_000_000 / 2), // TODO: Is the value correct?
+            RtcFastClock::RtcFastClockXtalD2 => HertzU32::Hz(40_000_000 / 2), /* TODO: Is the value correct? */
```

Comment:
Is there any reason for this change? `/*...*/` is discouraged in the guidelines

Reply:
good eyes, thanks, my rust analyzer/rustfmt decided for this, I'll see how I can encourage it not to.

Reply:
A valid point, but I'd rather not wait for a full CI cycle on this PR. At the very least this is internal and we can address later (I think we even found a rustfmt option to this for us?)

Reply:
Too late :) Looks like rustfmt prefers C-style comments when the line is too long.